flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/batch/table/CalcITCase.scala

```diff
@@ -217,6 +218,65 @@ class CalcITCase(
   }
 
   @Test
+  def testDisjunctivePredicateEarlyOut(): Unit = {
```

Comment:
Can you convert this test to a unit test with `ExpressionTestBase`? You can add them to `ScalarOperatorsTest`.